### DISCUSSION TITLE
How to set default embedded image size ?

### DISCUSSION THREAD

**Discussion by olcariz** [zsviczian/obsidian-excalidraw-plugin#1491] [Q&A]
How to set default embedded image size ? In my case to automatically adjust to page width.

I didn't find any setting for that, but i saw a previous setting in video for the default width, but nothing in current settings. Putting 100% in the backlink everytime is a bit of a hustle.

**Comment by timmyomahony** [comment]
This is an old question, but I was able to fix with a custom CSS file. Create a new file `excalidraw-fullwidth.css` and put the following in it:

```
.excalidraw-svg,
.excalidraw-embedded-img {
    width: 100%;
    max-width: 100%;
}
```

Now open settings and go to "Appearance". Down the bottom should be the "CSS Snippets" section. Click the little folder in the top right of this section to open the folder and now drop the new CSS file you created in there.

Back in Obsidian, hit the refresh and enable the new snippet. Embedded Excalidraw drawings should now be full-width
